src/main/java/fruitshop/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.strategy;
+
+import fruitshop.model.Operation;
+import fruitshop.strategy.operation.OperationService;
+import fruitshop.strategy.operation.operationimpl.BalanceOperation;
+import fruitshop.strategy.operation.operationimpl.PurchaseOperation;
+import fruitshop.strategy.operation.operationimpl.ReturnOperation;
+import fruitshop.strategy.operation.operationimpl.SupplyOperation;
+
+public class OperationStrategy {
+    public OperationService getCertainOperation(Operation operation) {

```

Comment:
Don’t use a switch-case here. Just pass Map<Operation, OperationHandler> with the relevant realization from Main.